File: Arrays/Java/LargestElement.java
import java.util.*;

class Largest {
    public int largest(List<Integer> arr, int n) {
        int max = arr.get(0);
        for (int i = 1; i < n; i++) {
            if (arr.get(i) > max) {
                max = arr.get(i);
            }
        }
        return max;
    }
}

public class LargestElement {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        List<Integer> arr = new ArrayList<>();

        System.out.println("Enter the number of elements in array:");
        int n = sc.nextInt();

        System.out.println("Enter the elements of array:");
        for (int i = 0; i < n; i++) {
            arr.add(sc.nextInt());
        }

        Largest l = new Largest();
        int result = l.largest(arr, n);
        System.out.println("The largest element is -> " + result);

        sc.close();
    }
}

File: Arrays/C++/easy/LargestElement.cpp
#include<iostream>
#include<vector>

using namespace std;

class Largest{
    public:
        int largest(const vector<int>arr,int n){
            int max=arr[0];
            for(int i=1;i<n;i++){
                if(arr[i]>max){
                    max=arr[i];
                }
            }
            return max;
        }
};
int main(){
    vector<int>arr;
    int n,temp;
    cout<<"Enter the no of elements in array"<<endl;
    cin>>n;
    cout<<"Enter the elements of array"<<endl;
    for(int i=0;i<n;i++){
        cin>>temp;
        arr.push_back(temp);
    }
    Largest l;
    int r=l.largest(arr,n);
    cout<<"the largest element is -> "<<r<<endl;
    return 0;
}